### DISCUSSION TITLE
P2P模式部署后，添加合作节点失败

### DISCUSSION THREAD

**Discussion by 1998zxn** [secretflow/secretflow#1518] [Q&A] [answered]
使用P2P方式部署两台节点后，添加合作节点需要填写所属机构与所属机构ID，然而并没有获取到该信息。
![添加合作节点](https://github.com/user-attachments/assets/fdf97d8c-c307-4ebf-b257-f0b45e189d3a)
部署命令：
![部署命令](https://github.com/user-attachments/assets/b53d9a88-5742-4053-a4f2-a9343961c994)

**Comment by wangzul** [comment, marked as answer]
关于添加合作节点可以参考[文档](https://www.secretflow.org.cn/zh-CN/docs/secretpad-all-in-one/v1.9.0b2/p2p_deploy/cooperation_node)

**Reply by 1998zxn** [reply]
我也看过了这个文档，在参考文档里，添加合作节点并不需要填写所属机构及ID的信息。
![参考文档](https://github.com/user-attachments/assets/5f455abb-d026-49a0-a321-3ecc480cc5ff)

**Comment by wangzul** [comment]
![image](https://github.com/user-attachments/assets/f687d830-aa63-42ce-8d77-3597f629c025)

添加合作节点时需要从合作方机构获取，你可以建立一个新节点配合完成合作节点的添加。

**Reply by 1998zxn** [reply]
感谢！！！！！我复制输入了节点认证码后，自动解析出了全部信息。
![节点认证码](https://github.com/user-attachments/assets/722779da-ba2d-44e7-a664-151a02c45d44)

**Reply by wangzul** [reply]
> 感谢！！！！！我复制输入了节点认证码后，自动解析出了全部信息。 ![节点认证码](https://private-user-images.githubusercontent.com/61788778/374421182-722779da-ba2d-44e7-a664-151a02c45d44.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgzNjYxNDYsIm5iZiI6MTcyODM2NTg0NiwicGF0aCI6Ii82MTc4ODc3OC8zNzQ0MjExODItNzIyNzc5ZGEtYmEyZC00NGU3LWE2NjQtMTUxYTAyYzQ1ZDQ0LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA4VDA1MzcyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA2NjhmZDEyMjgwMjc4MTUzZTc5MjE2NTQwNjgyYzQ2MzE3ODFkNDhkOWNmYTFiNTA2YzFjN2Q5NDQ1YjQ4NDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jtlWV76zroCWH_nCHPih4fuAiZP3HXZCtW7ScD-xkPA)

注意：节点信息未做改动的情况下，解析出来的通讯地址默认为127.0.0.1你需要确认添加合作的节点是否能通过该地址访问的被合作方，如果不能需要手动修改。
